requirements/extras/sqs.txt

```diff
@@ -1 +1,3 @@
-kombu[sqs]
+boto3>=1.9.12
+pycurl==7.43.0.2
+urllib3>=1.26.7 # Unittests are failing with urllib3>=1.28
```

Comment:
Not sure about this urllib3 dependency. I grabbed it from https://github.com/celery/kombu/commit/f3b04558fa0df4ecc383c93e0b3b300d95e17c47

The comment about version 1.28 and above breaking no longer makes much sense but from what I can see no 1.28 version exists for urllib3, see https://pypi.org/project/urllib3/#history 🤔